code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -46,29 +46,18 @@
 
 /obj/effect/spawner/lootdrop/armory_contraband
 	name = "armory contraband gun spawner"
+	lootcount = 3
 	lootdoubles = FALSE
-
 	loot = list(
-				/obj/item/gun/ballistic/automatic/pistol = 8,
-				/obj/item/gun/ballistic/shotgun/automatic/combat = 5,
-				/obj/item/gun/ballistic/automatic/pistol/deagle,
-				/obj/item/gun/ballistic/revolver/mateba
+				/obj/item/grenade/iedcasing,
+				/obj/item/grenade/smokebomb,
+				/obj/item/poster/random_contraband,
+				/obj/item/storage/fancy/cigarettes/cigpack_syndicate,
+				/obj/item/storage/fancy/cigarettes/cigpack_mindbreaker,
+				/obj/item/melee/baton/cattleprod,
+				/obj/item/toy/cards/deck/syndicate,

```

Comment:
this shit doesn't belong in the ARMORY

Reply:
it isnt supposed to be usefull you uncivilized powercreeper its a CONTRABRAND locker for contrabrand not for fucking guns you already have the whole fucking armoury